src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+    private static final String RESULT_LOG_STRING = "result";
+    private static final String ROW_SEPARATOR = ",";
+    private static final int DATA_INDEX_OPERATION_TYPE = 0;
+    private static final int DATA_INDEX_AMOUNT = 1;
+    private int supply = 0;
+    private int buy = 0;
+    private String fromFileName;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        this.fromFileName = fromFileName;
```

Comment:
it's better to pass this variable to the method, not as an object variable

Reply:
Understood.